README.md

```diff
@@ -28,7 +28,7 @@ Come on in, the water's lovely. More help? Ping `jfire` or `bhousel` on
 
 ## Prerequisites
 
-* [Node.js](http://nodejs.org/) version 4 or newer
+* [Node.js](http://nodejs.org/) version 4 or newer, ([npm](https://www.npmjs.com/package/npm) version 3 or newer)
 * Command line development tools (`make`, `git`, and a compiler) for your platform
```

Comment:
hmm, we probably did need it for something at one point (osmium?), but we're really close to dropping this entire bullet point.  Still using `make` for a few things, but it can be dropped.